accounts/src/inmemory_account_store.rs

```diff
@@ -260,14 +263,15 @@ impl AccountStorageInterface for InmemoryAccountStore {
                 if let Some(prev_account) = prev_account {
```

Comment:
instead of **Occupied, etc**, this should do it:
`self.owner_map_accounts.entry(owner).or_default().insert(account);`

Reply:
We do different things if account space has been occupied or not.